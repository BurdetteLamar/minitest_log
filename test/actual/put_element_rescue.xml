<log>
  <summary verdicts='0' failures='0' errors='1'/>
  <element>
    <rescued_exception>
      <class>
        RuntimeError
      </class>
      <message>
        Rescue me!
      </message>
      <backtrace/>
    </rescued_exception>
  </element>
  <element>
    <uncaught_exception timestamp='0000-00-00-xxx-00.00.00.000' class='RuntimeError'>
      <message>
        Not rescued!
      </message>
      <backtrace>
      </backtrace>
    </uncaught_exception>
  </element>
</log>
