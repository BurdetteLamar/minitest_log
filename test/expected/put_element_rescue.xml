<log>
  <summary verdicts='0' failures='0' errors='1'/>
  <rescue>
    <rescued_exception class='RuntimeError' message='Rescued!'>
      <backtrace/>
    </rescued_exception>
  </rescue>
  <no_rescue>
    <uncaught_exception timestamp='0000-00-00-xxx-00.00.00.000' class='RuntimeError'>
      <message>
        Not rescued!
      </message>
      <backtrace>
      </backtrace>
    </uncaught_exception>
  </no_rescue>
</log>
